<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: black;
    color: white;
}
/* Customize QPushButton specifically */
QPushButton {
    background-color: gray; /* A slightly lighter background for buttons */
    color: white;
    border: 2px solid white;
    border-radius: 5px;
}

/* Customize QTabWidget */
QTabWidget::pane {
    background-color: black;
    border: 1px solid white;
}

QTabWidget::tab-bar {
    alignment: center;
}

QTabBar::tab {
    background: gray;
    color: white;
    padding: 5px;
    border: 1px solid white;
}

QTabBar::tab:selected {
    background: white;
    color: black;
}

QLineEdit {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 2px;
}
font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Login">
       <attribute name="title">
        <string>CLogin</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>191</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLine">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passLine">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
          <widget class="QPushButton" name="loginButto">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QLabel" name="resultLabel">
           <property name="text">
            <string>Awaiting Login ...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Register">
       <attribute name="title">
        <string>Register</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>281</width>
          <height>341</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="regUser"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="regPass"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Liquor License Number:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="regLnum"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Liquor License Expiry Year:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="regLdate"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Business Adderess</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="regAdrs"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Credit Card Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="regCard"/>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="regName"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="regButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>130</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
       <widget class="QLabel" name="regResult">
        <property name="geometry">
         <rect>
          <x>406</x>
          <y>160</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Result...</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="MLogin">
       <attribute name="title">
        <string>MLogin</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>139</x>
          <y>0</y>
          <width>221</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mUser"/>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mP1">
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Second Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mP2">
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QPushButton" name="mLoginButton">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mResult">
           <property name="text">
            <string>Result...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEcomLiq">
    <property name="title">
     <string>EcomLiq</string>
    </property>
   </widget>
   <addaction name="menuEcomLiq"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
